include/ck/tensor_operation/gpu/device/device_add_normalization.hpp

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2022, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include <iostream>
+#include <vector>
+
+#include "ck/tensor_operation/gpu/device/device_base.hpp"
+
+namespace ck {
+namespace tensor_operation {
+namespace device {
+
+template <typename ADataType,
+          typename BDateType,
+          typename CDataType,
+          typename GammaDataType,
+          typename BetaDataType,
+          typename AccDataType,
+          typename YDataType,
+          typename ElementwiseOperation,

```

Comment:
It is better to define the naming of input of layernorm.
Naming of ElementwiseOperation make reader confuse which tensors would be applied by this lambda